README.md

```diff
@@ -2,30 +2,77 @@
 
 An index of applications built for the nRF Connect SDK.
 
+----
+
 ## Contributing
 
-If you or your organization has a project that you'd like to include in the nRF Connect SDK App Index, you can open a pull request against this repository. See the details shown in the [`CONTRIBUTING`](./CONTRIBUTING.md) file for more information.
+If you or your organization has a project that you'd like to include in the nRF Connect SDK App Index, you can open a pull request against this repository. Read the [`CONTRIBUTING`](./CONTRIBUTING.md) file for more information.
+
+----
 
 ## Development
 
-To build the index file, run `npm run generate-index-json` from the root directory.
+To generate a new `index.json` file and use it on a custom version of the App Indes page, complete the following steps:
 
-### Website
+1. Generate the `index.json` file:
 
-This repository includes a static website that displays the contributed applications in a searchable frontend. It's developed using the [Next.js](https://nextjs.org/) React framework, and uses [Tailwind](https://tailwindcss.com/) for styling.
+   a. Open a command terminal in the `ncs-app-index` root directory.
 
-To launch a live server development server, run `npm run dev` inside the `site` directory. By default, the site runs on port 4000, but this can be overridden by setting the `PORT` environment variable.
+   b. Install the required dependencies:
+      ```
+      npm install
+      ```
 
-Application data is read at build time. For development purposes, sample data can be imported from `sampleData.ts`.
+   c. Generate `index.json`:
+      ```
+      npm run generate-index-json
+      ```
+      The `index.json` is created in the `resources` directory.
+
+   d. Copy `index.json` to `./site/public/`.
+
+1. Serve the website locally:
+
+   a. Go to the `site` directory.
+
+   b. Install the required dependencies:
+      ```
+      npm install
+      ```
+
+   c. Build the website locally:
+      ```
+      npm run build
+      ```
+      This will create a static build in the `/site/out` directory.
 
-#### Building
+   d. Start the website locally:
+      ```
+      npm run dev
+      ```
 
-To build the website, run the following commands:
+1. Check if the site works:
 
-1. In the root directory, run `npm install`
-2. In the `site` directory, run `npm run build`
+   a. Access the site in the browser by going to `localhost:4000/ncs-app-index`.
 
-This will create an optimised static build in the `/site/out` directory.
+   b. Check if the `index.json` is available by going to `localhost:4000/ncs-app-index/index.json`.
+
+1. Set the [nRF Connect for Visual Studio Code extension](https://nrfconnect.github.io/vscode-nrf-connect/reference/ui_sidebar_welcome.html#create-a-new-application) to fetch the application index from the custom URL:
```

Comment:
Perhaps the 1st step is to modify the settings, then the 2nd step could instruct to test the index by creating an application from the app index.

Reply:
Rephrased into sections.